### PR TITLE
Feb issue v8

### PR DESCRIPTION
**Pull request by rishi9684**
#issue resolved
#video console error removed

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -66,13 +66,13 @@ export default {
...
           'https://cdnjs.cloudflare.com/ajax/libs/geopattern/1.2.3/js/geopattern.min.js',
       },
-      {
-        src: 'https://unpkg.com/video.js/dist/video.js',
-      },
-      {
-        src:
-          'https://unpkg.com/videojs-contrib-hls/dist/videojs-contrib-hls.js',
-      },
+      // {
```
Comment (SukantGujar) [inline nuxt.config.js]: Remove

File: pages/apps/_app/attendee/_id/index.vue [hunk 2]
```diff
@@ -1144,6 +1151,20 @@ export default {
...
     },
+    checkLiveSession(item) {
+      const liveStart =
+        new Date().getTime() > new Date(item.StartDate).getTime()
+      const liveEnd =
+        new Date().getTime() <
+        new Date(
+          new Date(item.StartDate).getTime() + item.Duration * 60000
+        ).getTime()
+      if (liveStart && liveEnd) {
```
Comment (SukantGujar) [inline pages/apps/_app/attendee/_id/index.vue:1162]: Just `return liveStart && liveEnd;`

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: pages/apps/_app/attendee/_id/index.vue (hunk 2 above)
Comment (rishi9684) [reply pages/apps/_app/attendee/_id/index.vue:1162]: done

File: pages/apps/_app/attendee/_id/index.vue [hunk 3]
```diff
@@ -1144,6 +1151,18 @@ export default {
...
```
Comment (SukantGujar) [inline pages/apps/_app/attendee/_id/index.vue:1162]: No need for if, just return.

**Review by SukantGujar — CHANGES_REQUESTED** [review]

**Review by SukantGujar — APPROVED** [review]

**Pull request merged**